Gemfile

```diff
@@ -1,3 +1,3 @@
 source 'https://rubygems.org' do
-  gem 'cocoapods', '~> 1.4'
+  gem 'cocoapods', '~> 1.6'
```

Comment:
Should this be 1.7?  They just released 1.7.0

Reply:
It should! This PR is to unblock https://github.com/square/Valet/pull/174, which bumps to 1.7. CP 1.4 didn’t work on Xcode 10.2 CI: 1.6 was the earliest release that did.